src/Symfony/Component/Debug/Exception/FlattenException.php

```diff
@@ -324,4 +324,51 @@ private function getClassNameFromIncomplete(\__PHP_Incomplete_Class $value)
 
         return $array['__PHP_Incomplete_Class_Name'];
     }
+
+    public function getTraceAsString()
+    {
+        $trace = '';
+
+        $items = $this->trace;
+        array_shift($items);
+
+        $i = -1;
+
+        foreach ($items as $i => $item) {
+            $trace .= '#' . $i . ' ';
+            if (!empty($item['file'])) {
+                $trace .= $item['file'].'('.$item['line'].'): ';
+            }
+
+            if (!empty($item['function'])) {
+                if (!empty($item['class'])) {
+                    $trace .=  $item['class']  . $item['type'];
+                }
+                $trace .= $item['function'];
+                $trace .= self::traceArgumentsToString(
+                    !empty($item['args']) ? $item['args'] : []
+                );
+            }
+
+            $trace .= "\n";
+        }
+
+        $i++;
+
+        $trace .= "#". $i . " {main}";
+
+        return $trace;
+    }
+
+    public function __toString()
+    {
+        $message = '';
+
+        foreach (array_merge(array($this), $this->getAllPrevious()) as $exception) {
+            $message .= $exception->getClass() . ' in ' . $exception->getFile() . ':' . $exception->getLine() .
```

Comment:
Shouldn't it also include the exception message here (your tests are using an empty message, but most exceptions have a non-empty one)